From: Alexis Murzeau <amubtdx@gmail.com>
Date: Fri, 29 Dec 2017 00:41:30 +0100
Subject: Remove use of docs/_static directory

This directory is removed from the source package for dfsg-compatibility
reasons.
This patch remove the use of that directory to avoid warnings from sphinx.

Forwarded: https://github.com/streamlink/streamlink/pull/1395
Author: Alexis Murzeau <amubtdx@gmail.com>
---
 docs/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index a8b1345..2b230bc 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -129,7 +129,7 @@ html_theme_options = {
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+#html_static_path = ['_static']
 
 # If not '', a 'Last updated on:' timestamp is inserted at every page bottom,
 # using the given strftime format.
